### ISSUE TITLE
Create references from Incoming References pane

### ISSUE THREAD

**Issue by geball** [sanity-io/sanity-plugin-documents-pane#4]
Feature request: It would be handy to be able to create references directly from the Incoming References pane.

**Comment by dpfavand** [comment]
I would like to make this happen. Here's a suggested approach:

1. Add a configuration item, `initialValueTemplates`, that accepts a function. That function returns an array of initial value template configurations. The function receives the `SanityDocument` so it can be responsive to values on the draft or published document.
2. Render a button above the list for each initial value template that is registered. When the user clicks the button, open a pane to the right using the initial value template.

Config example:
```
S.view
  .component(DocumentsPane)
  .options({
    query: `*[_type == 'post' && author._ref == $id]`,
    params: {id: `_id`},
    useDraft: false,
    debug: true,
    initialValueTemplates: (document) => {
       
        const id = document._id.replace('drafts.', '')

        return [
          {
             title: 'New post for author',
             template: 'postWithAuthor',
             params: {
               authorId: document._id
             }
          }
        ]

    }
  })
  .title('Posts')

```

**Comment by dpfavand** [comment]
Closed by #11
